bigtop-packages/src/charm/hadoop/layer-hadoop-resourcemanager/actions/smoke-test

```diff
@@ -43,6 +43,6 @@ smoke_env = {
 bigtop = Bigtop()
 result = bigtop.run_smoke_tests(smoke_components, smoke_env)
 if result == 'success':
-    hookenv.action_set({'outcome': result})
+    hookenv.action_set({'outcome': 'success'})
 else:
-    fail('{} smoke tests failed'.format(smoke_components), result)
+    fail('{} smoke tests failed with: {}'.format(smoke_components, result))
```

Comment:
@johnsca, I merged the output with the message to be consistent with other bigtop charms.  When an action fails, users expect to see the reason in the `message` key.  Without this change, they would see the abbreviated failure message and need to know to look elsewhere for an `output` key.